### ISSUE TITLE
Bing Web Search : "body is not defined"

### ISSUE THREAD

**Issue by RomainSimon** [joshbalfour/node-cognitive-services#4]
I tried to use Bing Web Search as mentionned in the example : https://www.npmjs.com/package/cognitive-services#bing-web-search

I removed the "new" causing the error "cognitiveServices.BingWebSearch  is not a constructor" as mentionned here : https://github.com/joshbalfour/node-cognitive-services/issues/2

```
const bingWebSearch = new cognitiveServices.bingWebSearch({
    API_KEY: yourApiKey
})
 
const parameters = {
    q: "bill gates"
    count: "10"
    offset: "0"
    mkt: "en-us"
    safesearch: "Moderate"
};
 
bingWebSearch.search({
        parameters
    })
    .then((response) => {
        console.log('Got response', response);
    })
    .catch((err) => {
        console.error('Encountered error making request:', err);
    });
 
```

All I get is a "body is not defined" on BingWebSearch.js:110

Any help ?

Thanks !

**Comment by RomainSimon** [comment]
For those having the same problem, I used [node-bing-api](https://www.npmjs.com/package/node-bing-api) that supports Cognitive Services instead

**Comment by miparnisari** [comment]
@RomainSimon this is fixed in version 0.2.0
